### PR TITLE
 Improve types for web workers

### PR DESCRIPTION
**Pull request by bjornstar**
We can leverage the `WebWorker` types for the `webworkers` directory.

I couldn't help touching the `animation_frame` to remove a branch that would only occur on the first frame. If you would prefer to leave it as is, I'd be happy to back that out.

### CODE REVIEW

File: src/js/core/animation_frame.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import { Signal } from "./signal";
 
-// @ts-ignore
-import BackgroundAnimationFrameEmitterWorker from "../webworkers/background_animation_frame_emittter.worker";
+import BackgroundAnimationFrameEmitterWorker from "worker-loader!../webworkers/background_animation_frame_emittter.worker";
```
Comment (tobspr) [inline src/js/core/animation_frame.js:3]: I think you need to pass the `fallback` and `inline` options here then, not?

File: src/js/core/async_compression.js [hunk 2]
```diff
@@ -1,7 +1,6 @@
-// @ts-ignore
-import CompressionWorker from "../webworkers/compression.worker";
+import CompressionWorker from "worker-loader!../webworkers/compression.worker";
```
Comment (tobspr) [inline src/js/core/async_compression.js:1]: Same here

File: src/js/core/animation_frame.js (hunk 1 above)
Comment (bjornstar) [reply src/js/core/animation_frame.js]: Sorry about that, for some reason I read their docs as inline being the default.

**Comment by bjornstar** [comment]
I cherry-picked my fix for the trailing spaces from #504

**Comment by tobspr** [comment]
ty!

**Pull request merged**